pyproject.toml

```
@@ -70,6 +70,7 @@ sphinx-copybutton = {version = ">=0.3.1,<0.6.0", optional = true}
 myst-parser = {version = ">=0.17.2,<1.1.0", optional = true}
 sphinx-autobuild = {version = "^2021.3.14", optional = true}
 sphinx-reredirects = {version = "^0.1.1", optional = true}
+sphinx-inline-tabs = {version = "^2023.4.21"}
```

To make sure this isn't pulled every time someones installs the Singer SDK:

```suggestion
sphinx-inline-tabs = {version = "^2023.4.21", optional = true}
```

You'll also have to run `poetry lock` and push the updated `poetry.lock` file.